src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -62,6 +64,7 @@ export const RoomModal: React.FC<RoomModalProps> = ({
           room={room}
           venueName={venue.name}
           venueEvents={venueEvents}
+          venueTemplate={venue.template ?? ""}
           showSchedule={venue.showSchedule}
```

Comment:
pass a venue there and take the values there

Reply:
fixed in 0d120bdff6a492683655d4e6d3db09a76b61a811 15328f6daf4da0100e94cbe9250e999ab57b972c

Reply:
@mike-lvov